src/import/parserContexts.ts

```diff
@@ -15,6 +15,7 @@ export interface EntityContext extends ParserRuleContext {
   ruleSet(): RuleSetContext;
   paramRuleSet(): ParamRuleSetContext;
   mapping(): MappingContext;
+  logical(): LogicalContext;
```

Comment:
We need `resource()` here too:
```suggestion
  logical(): LogicalContext;
  resource(): ResourceContext;
```